### PR TITLE
Docker desktop 4.28.0 release notes

### PR DESCRIPTION
**Pull request by lorenrh**
Docker Desktop 4.28.0 release notes

### CODE REVIEW

File: content/desktop/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,57 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+    - wasmedge `v0.13.5`
+    - wasmer `v4.1.2`
+  - Updated deislabs shims to `v0.11.1`, including:
+    - lunatic `v0.13.2`
+    - slight `v0.5.1`
+    - spin `v2.2.0`
+    - wws `v1.7.0`
+
+### Bug fixes and enhancements
+
```
[the remark this answers is not in the document]
Comment (lorenrh) [reply content/desktop/release-notes.md:51]: Yes! and I think that should be in the new section, must've missed it when gathering the release notes. Will update!

File: content/desktop/release-notes.md [hunk 2]
```diff
@@ -24,6 +24,66 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Allow `SOCKS` proxies to be set via a proxy autoconfig `pac file`.
+- Re added `CONFIG_AUDIT` kernel config.
+- Fixed a bug with the Rust build on `virtiofs`. See [rust-lang/docker-rust#161](https://github.com/rust-lang/docker-rust/issues/161).
+- Fixed an issue that caused the `missing registry authentication` error when pulling Kubernetes images.
+- Fixed an issue that caused Docker Compose commands to hang.
+- Fixed a bug in `docker build` that caused Docker Desktop to crash. Fixes [docker/for-win#13885](https://github.com/docker/for-win/issues/13885), [docker/for-win#13896](https://github.com/docker/for-win/issues/13896), [docker/for-win#13899](https://github.com/docker/for-win/issues/13899), [docker/for-mac#7164](https://github.com/docker/for-mac/issues/7164), [docker/for-mac#7169](https://github.com/docker/for-mac/issues/7169)
+- Build UI:
+  - Active and completed builds can be found in dedicated tabs.
+  - Build details now displays build duration and cache steps.
+  - OpenTelemetry traces are now displayed the build results.
```
Comment (crazy-max) [inline content/desktop/release-notes.md:71]:
```suggestion
  - OpenTelemetry traces are now displayed in the build results.
```

File: content/desktop/release-notes.md [hunk 3]
```diff
@@ -24,6 +24,66 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fix `httpd` issue with Rosetta. [docker/for-mac#7182](https://github.com/docker/for-mac/issues/7182)
+- Fixed a bug that caused a crash on the `virtualization.framework`. Fixes [docker/for-mac#7024](https://github.com/docker/for-mac/issues/7024)
+
+#### For Windows
+
+- Fixed an issue with DNS timeouts on Windows.
+- Added support for Enhanced Container Isolation Docker socket mount permission on WSL user distros.
+- Fixed an issue that caused the `failed to get console mode` error when redirecting output from the CLI.
+- Fixed an issue with the engine socket permissions when mounted inside containers. Fixes [docker/for-win#13898](https://github.com/docker/for-win/issues/13898)
+
```
[the remark this answers is not in the document]
Comment (lorenrh) [reply content/desktop/release-notes.md:86]: Makes sense :)

File: content/desktop/release-notes.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lorenrh) [reply content/desktop/release-notes.md:57]: At the end we are not mentioning rhel support, until subsequent releases

**Pull request merged**